timetable.md

```diff
@@ -29,3 +29,27 @@
 
 - **75** [Exercise: SSE Git cheat sheet](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/version-control/material/cheat_sheet_text.md)
 - **15** Discussion
+
+## 3.1 -- Thu, November 04, 2021
+
+- **5** [Virtualization and containers](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/intro_slides.md)
+- **10** [Introduction to virtualization](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/virtualmachines_slides.md)
+- **40** [VirtualBox (including practical examples)](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/virtualbox_slides.md)
+- **25** [Vagrant (including practical examples)](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/vagrant_slides.md)
+- **10** [Introduction to Containers](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/containers_slides.md)
+
+- Tell students to get/install virtualbox in case they have a machine without Linux and or without root rights.
+
+## 3.2 -- Thu, November 04, 2021
+
+- **5** [Docker quiz: "What is Docker?"](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/docker_quiz.md)
+- **65** [Docker (including practical examples)](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/docker_slides.md)
+- **20** [Singularity (including practical examples)](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/virtualization-and-containers/material/singularity_slides.md)
+
```

Comment:
Another idea: you could show the learning goals in the very end again and let students give answers?

Reply:
I think that is a good idea. I will put it on the timeline with a note to show the intro slides again.